src/test_l3_dsCompositeIn.c

```diff
@@ -0,0 +1,480 @@
+
+    gConnectionStatus = isPortConnected;
+}
+
+/**
+ * @brief Callback function for CompositeIn signal change.
+ *
+ * This function is invoked whenever a signal change occurs in the Comosite Input connection.
+ */
+static void compositeInSignalChangeCB(dsCompositeInPort_t port, dsCompInSignalStatus_t sigStatus)
+{
+    UT_LOG_INFO("Received SignalChange status callback port: %s, sigstatus: %s",
+                 UT_Control_GetMapString(dsCompositeInPortMappingTable, port),
+                 UT_Control_GetMapString(dsCompInSignalStatusMappingTable, sigStatus));
+
+    gSignalStatus = sigStatus;
+}
+
+/**
+ * @brief Callback function for CompositeIn status change.
+ *
+ * This function is invoked whenever a status change occurs in the CompositeIn connection.
+ */
+static void compositeInStatusChangeCB(dsCompositeInStatus_t inputStatus)
+{
+    UT_LOG_INFO("Received statuschange callback isPresented: %s, activeport: %s",
+                 UT_Control_GetMapString(bool_mapTable, inputStatus.isPresented),
+                 UT_Control_GetMapString(dsCompositeInPortMappingTable, inputStatus.activePort));
+
+    for(int i = 0 ; i < dsCOMPOSITE_IN_PORT_MAX ; i++) 
+    {
+         if(!(inputStatus.isPresented))
+                 continue;
+
+         UT_LOG_INFO("Received statuschange callback isPortConnected: %s, activeport: %s",
+                 UT_Control_GetMapString(bool_mapTable, inputStatus.isPortConnected[i]),
+                 UT_Control_GetMapString(dsCompositeInPortMappingTable, inputStatus.activePort));
+    }
+
+    gStatusChange = inputStatus;
+}
+
+/**
+* @brief This test initializes the CompositeIn Module.
+*
+* This test function initializes the CompositeIn Module.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 001@n
+*
+*/
+void test_l3_CompositeIn_initialize(void)
+{
+    gTestID = 1;
+    UT_LOG_INFO("In %s [%02d%03d]", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret = dsERR_NONE;
+
+    /* Initialize the CompositeIn Module */
+    UT_LOG_INFO("Calling dsCompositeInInit()");
+    ret = dsCompositeInInit(); 
+    UT_LOG_INFO("Result dsCompositeInInit() dsError_t:[%s]", UT_Control_GetMapString(dsError_mapTable, ret));
+    ASSERT(ret == dsERR_NONE);
+
+    /* Register connection status callback */
+    UT_LOG_INFO("Calling dsCompositeInRegisterConnectCB(CBFunc:[0x%0X])", compositeInConnectCB);
+    ret = dsCompositeInRegisterConnectCB(compositeInConnectCB);
+    UT_LOG_INFO("Result dsCompositeInRegisterConnectCB(CBFunc:[0x%0X]) dsError_t:[%s]",
+                        compositeInConnectCB, UT_Control_GetMapString(dsError_mapTable, ret));
+    ASSERT(ret == dsERR_NONE);
+
+    /* Register Signal change callback */
+    UT_LOG_INFO("Calling dsCompositeInRegisterSignalChangeCB(cbFun:[0x%0X])", compositeInSignalChangeCB);
+    ret = dsCompositeInRegisterSignalChangeCB(compositeInSignalChangeCB);
+    UT_LOG_INFO("Result dsCompositeInRegisterSignalChangeCB(cbFun:[0x%0X]) dsError_t:[%s]",
```

Comment:
cbFun , CBFunc we can have one common name. In some places, we are using CBFunc and some places named as  cbFun

Reply:
done